#goal_definition
---
#result definition
bool done
---
#feedback
float64 error
